packages/emoji-mart/package.json

```diff
@@ -32,11 +30,15 @@
     "prepublishOnly": "yarn build"
   },
   "devDependencies": {
-    "@parcel/transformer-inline-string": "2.3.2",
-    "@parcel/transformer-sass": "2.3.2",
+    "@parcel/packager-ts": "2.5.0",
+    "@parcel/transformer-inline-string": "2.5.0",
+    "@parcel/transformer-sass": "2.5.0",
+    "@parcel/transformer-typescript-types": "2.5.0",
     "autoprefixer": "10.4.4",
-    "parcel": "2.3.2",
-    "preact": "10.6.4"
+    "parcel": "2.5.0",
+    "postcss": "8.4.13",
+    "preact": "10.6.4",
+    "typescript": "4.6.4"
```

Comment:
The dependencies I added so far were required, otherwise it was throwing a "{module} not found" error. I don't know why it is throwing for `postcss` though. I will try to remove it and/or figure out why it is required.

Reply:
No worries. I’m still experimenting / figuring out the whole workspaces and the best way to configure everything. Are you running scripts from the root or from `packages/emoji-mart`? If the latter, that might be why (and also maybe why you got around the typescript issue from #576). With workspaces, you need to handle dependencies from the root. Dependencies can still be added to a specific workspace (like it currently is), but there’s only one `node_modules` at the root. Workspaces install all workspaces dependencies at the root.

We could even move all `devDependencies` to the root `package.json` if that helps with issues.